src/public/media.ts

```diff
@@ -226,22 +228,30 @@ export namespace media {
     maxMediaCount: number;
 
     /**
-     * Additional properties for customization of select media in mobile devices
+     * Additional properties for customization of select media - Image in mobile devices
      */
     imageProps?: ImageProps;
 
+    /**
+     * Additional properties for customization of select media - VideoAndImage in mobile devices
+     */
+    videoAndImageProps?: VideoAndImageProps;
+
     /**
      * Additional properties for audio capture flows.
      */
     audioProps?: AudioProps;
   }
 
   /**
-   *  All properties in ImageProps are optional and have default values in the platform
```

Comment:
why is this comment removed? It is still applicable I believe

Reply:
The comment is still on top of ImageProps (This is MediaProps where you are referring)